java/pub-sub-bot/src/main/java/com/google/chat/bot/pubsub/Main.java

```diff
@@ -41,129 +37,91 @@
  */
 public class Main {
 
-  public static final String CREDENTIALS_PATH_ENV_PROPERTY = "GOOGLE_APPLICATION_CREDENTIALS";
-
-  // Google Cloud Project ID
-  public static final String PROJECT_ID = "YOUR-PROJECT-ID";
+  public static void main(String[] args) {
+    System.out.println(System.getenv().toString());
```

Comment:
does need to be printed out? add an info log if valuable

Reply:
Oops, was just left over from some quick debugging :)